docker-app/qfieldcloud/core/admin.py

```diff
@@ -687,7 +704,10 @@ def created_by__link(self, instance):
 
     @admin.display(ordering="project__name")
     def project__name(self, instance):
-        return model_admin_url(instance.project, instance.project.name)
+        if instance.project:
+            return model_admin_url(instance.project, instance.project.name)
+
+        return "-"
```

Comment:
If you return `None` I think you should get the placeholder? I might be wrong too.